.storybook/main.js

```diff
@@ -23,6 +23,14 @@ module.exports = /** @type {Pick<StorybookConfig,'addons' |'stories' |'webpackFi
     '@storybook/addon-a11y',
     '@storybook/addon-knobs/preset',
     'storybook-addon-performance',
+    {
+      name: '@storybook/addon-postcss',
```

Comment:
q: why is this needed ?

Reply:
Since we require postcss `8.2.4`, this needed to be added per document snippet below:

![image](https://user-images.githubusercontent.com/8649804/123680382-7d129680-d7fd-11eb-8d2f-4747eb4dfa05.png)

Reply:
hmm I don't think we need postcss for vNext (as styling is processed by make-styles - so vendor prefix etc are being properly added - am I right @layershifter ?)

Reply:
Yes, it's correct, we don't need PostCSS for vNext components

Reply:
Got it - I will remove `addon-postcss` from the storybook addons and also remove the `postcss.config.js` file i created.